src/checker.ml

```diff
@@ -589,16 +597,20 @@ let entrypoint_add_liquidity (state, p: checker * (ctok * kit * lqt * Ligo.times
   let ctok_deposited, max_kit_deposited, min_lqt_minted, deadline = p in
   let (lqt_tokens, kit_tokens, updated_cfmm) =
     cfmm_add_liquidity state.cfmm ctok_deposited max_kit_deposited min_lqt_minted deadline in
+
   let transfer =
-    { address_from = !Ligo.Tezos.sender;
-      address_to = !Ligo.Tezos.self_address;
-      value = ctok_to_denomination_nat ctok_deposited;
+    { from_ = !Ligo.Tezos.sender;
+      txs = [
+        { to_ = !Ligo.Tezos.self_address;
+          token_id = ctok_token_id;
+          amount = ctok_to_denomination_nat ctok_deposited;
+        }
+      ];
     } in
   let op =
-    LigoOp.Tezos.fa12_transfer_transaction
-      transfer
-      (Ligo.tez_from_literal "0mutez")
-      (get_transfer_ctok_fa12_entrypoint state.external_contracts) in
+    LigoOp.Tezos.fa2_transfer_transaction
+      [transfer] (Ligo.tez_from_literal "0mutez")
+      (get_transfer_ctok_fa2_entrypoint state.external_contracts) in
```

Comment:
Random, but the diff of these changes from FA1.2 to FA2 transfers looks really nice and tidy.